Documentation/templates/strings/objc.md

```diff
@@ -0,0 +1,53 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File names | strings/objc-string-h.stencil & strings/objc-string-m.stencil |
+| Configuration example | <pre>strings:<br />  inputs: path/to/Localizable.strings<br />  outputs:<br />    - templateName: objc-string-h<br />      output: Localizable.h</pre><br />    - templateName: objc-string-m<br />      output: Localizable.m</pre> |
+| Language | Objective-C |
+| Author | Eric Slosser |
+
+## When to use it
+
+- When you need to generate *Objective-C* code.
```

Comment:
Could someone remind me when this template is more useful compared to generate Swift code compatible with ObjC (for which we have a PR too I think?) — with `@objcMembers class L10n: NSObject` instead of `enum` for namespacing so that they are visible to ObjC?

Is it for projects that only have ObjC code and no Swift at all in them yet, so you don't want to have a `Strings.swift` generated by SwiftGen to be the only Swift file and triggering the inclusion of the Swift runtime just for it?

Reply:
Essentially yeah.

Note that we still need to write some sort of (compilation) test for this template, but I'd leave that for another PR.

Reply:
Alright. We should mention that in the "When to use it" paragraph then, maybe?

Reply:
Depends. Do we expect to merge #490 as well as this PR? We can, but I don't think it's necessary.

Reply:
Do the Swift strings templates generate Swift code that can be used from ObjC? I don't see that `enum L10n` getting into "<projectname>-Swift.h".

Reply:
I'll add the following to `Documentation/templates/strings/objc.md`

> When you need to generate *Objective-C* code.  Perhaps because you have a pure Objective-C project and don't want to add a Swift file to it (thus triggering the inclusion of the Swift runtime into your build).  Or perhaps because you can't find a template that generates Swift that can be accessed from Objective-C.